website/source/guides/advanced-scheduling/spread.html.md

```diff
@@ -0,0 +1,226 @@
+---
+layout: "guides"
+page_title: "Spread"
+sidebar_current: "guides-advanced-scheduling"
+description: |-
+    The spread stanza allows operators to increase failure tolerance in their Nomad clusters by allowing them to distribute their workloads in a customized way based on attributes or client metadata.
+---
+
+# Increasing Failure Tolerance with Spread
+
+The Nomad scheduler uses a bin packing algorithm when making job placements on
+nodes to optimize resource utilization and density of applications. Although
+this feature ensures that cluster resources are being used efficiently, it does
```

Comment:
suggested rewording "Although
this feature ensures that cluster resources are being used efficiently," with "Although bin packing ensures optimal resource utilization, it can lead to some nodes carrying a majority of allocations for a given job. This can cause cascading failures where the failure of a single node or a single data center can lead to application unavailability"